android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -2265,6 +2327,7 @@ public ArrayList<MenuBottomSheetItem> getMenuBottomSheetItems(String id)
       if (!TextUtils.isEmpty(mDonatesUrl))
         items.add(new MenuBottomSheetItem(R.string.donate, R.drawable.ic_donate, this::onDonateOptionSelected));
       items.add(new MenuBottomSheetItem(R.string.settings, R.drawable.ic_settings, this::onSettingsOptionSelected));
+      items.add(new MenuBottomSheetItem(R.string.recent_track, R.drawable.ic_trace_path_off, -1, this::onTrackRecordingOptionSelected));
```

Comment:
Is badge count -1 a valid value?

Reply:
actually it sends the count to show in the badge infront of bottom sheet menu item. But for trace path we needed only a dot so i sent -1 as a flag to distinguish between feasible number.